packages/cli/src/options/beaconNodeOptions/network.ts

```diff
@@ -63,7 +120,7 @@ export function parseArgs(args: NetworkArgs): IBeaconNodeOptions["network"] {
       : null,
     maxPeers: maxPeers ?? defaultOptions.network.maxPeers,
     targetPeers: targetPeers ?? defaultOptions.network.targetPeers,
-    localMultiaddrs: [`/ip4/${listenAddress}/tcp/${tcpPort}`],
+    localMultiaddrs: [localMu, localMu6].filter(Boolean) as string[],
```

Comment:
libp2p was already configured to listen to an array of multiaddrs.
So nothing further than passing both multiaddrs here needs to be done.